Makefile

```diff
@@ -193,9 +207,9 @@ license-check:
 	@echo
 
 .PHONY: sanity-test
-sanity-test: sanity-test
+sanity-test: bootstrap
```

Comment:
it will not download if bootstrap tools are already installed. This has been added due to ginkgo. I'll remove the ginkgo and will go with `go test` only since timeout will set the definite limit for the test. If required we will increase it as per the need.